modules/graph_ql/main.tf

```diff
@@ -0,0 +1,72 @@
+
+data "aws_iam_policy_document" "graphql_task_policy_document" {
+  statement {
+    actions   = ["sts:AssumeRole"]
+    resources = ["*"]
+  }
+}
+
+data "aws_iam_policy_document" "assume_ecs_tasks_role_policy_document" {
+  statement {
+    principals {
+      type        = "Service"
+      identifiers = ["ecs-tasks.amazonaws.com"]
+    }
+    actions = ["sts:AssumeRole"]
```

Comment:
we should also update the policy to only allow required API calls in the future task.

Reply:
Yes only allow required API calls. 

Like you mentioned- 
`This role is required by tasks to pull container images and publish container logs to Amazon CloudWatch on your behalf.`
I was expecting to see such API calls in the role tf.

Reply:
since the image is being pushed to orca repo and not aws, we don't need much permissions for now.